terraform/vars/terraform-staging.tfvars

```diff
@@ -6,7 +6,8 @@ auto_scaling_min_capacity = 1
 auto_scaling_max_capacity = 15
 
 auth_url         = "https://staging-api.resourcewatch.org"
-forms_api_url             = "https://staging-api.resourcewatch.org/v1"
+forms_api_url    = "https://staging-api.resourcewatch.org/v1"
+alerts_api_url   = "https://staging-data-api.globalforestwatch.org/"
```

Comment:
Need to remove the ending `/` to avoid `//` issues with API calls
`"https://staging-data-api.globalforestwatch.org"`